docker-compose.yml

```diff
@@ -0,0 +1,62 @@
+version: "3.8"
+
+services:
+  theater_api:
+    build:
+      context: ./backend
+      dockerfile: ./build/Dockerfile
+      target: dev
+    image: theater/backend
+    container_name: theater_api
+    command:
+      - --reload
+    ports:
+      - "8015:5000"
+    volumes:
+      - ./backend:/opt/theater/backend
+    environment:
+      UVICORN_PORT: 5000
+      UVICORN_APP: src.main:app
+      UVICORN_WORKERS: 2
+    env_file:
+      - ./backend/.env.local
+    depends_on:
+      theater_db:
+        condition: service_healthy
+
+  theater_migration:
```

Comment:
Is it separate container for making migration easy, without going to container?

Don't you think that managing one extra container can be potentially overhead for server or it's just for development goals?

Reply:
It's just for development - we'll remove it once deploying to prod